macros/helpers/prime_line.cfg

```diff
@@ -30,6 +30,19 @@ gcode:
     {% set prime_line_y = params.START_Y|default(prime_line_y)|float %}
     {% set prime_line_direction = params.LINE_DIRECTION|default(printer["gcode_macro _USER_VARIABLES"].prime_line_direction)|string|upper %}
 
+    # Redefine start position from fl_size 
+    {% set fl_size=printer['gcode_macro START_PRINT'].fl_size %}
+    {% if fl_size != "0_0_0_0" %}
+        {% set prime_line_margin = params.LINE_MARGIN|default(printer["gcode_macro _USER_VARIABLES"].prime_line_margin|float) %}
+        {% set min_x, min_y, max_x, max_y = fl_size.split("_")|map('float') %}
+        {% set prime_line_x = [[prime_line_x, min_x - prime_line_margin] | max, max_x + prime_line_margin] | min %}
+        {% set prime_line_y = [[prime_line_y, min_y - prime_line_margin] | max, max_y + prime_line_margin] | min %}
```

Comment:
This verifies that x/y never goes under 0/min or over max?

Reply:
This ensure that prime line start point is inside FL_SIZE area + prime_line_margin.
The prime_line_way parameter attempt to avoid that prime_line leaves the print area in case of misconfiguration of prime line start point.